objects/src/accounts/seed.rs

```diff
@@ -0,0 +1,65 @@
+use super::{parse_felt, AccountError, AccountId, Hasher, Word};
+
+// ACCOUNT SEED GENERATOR
+// ================================================================================================
+
+/// Specifies the account type.
+pub(super) enum AccountType {
+    FungibleFaucet,
+    NonFungibleFaucet,
+    RegularAccountImmutableCode,
+    RegularAccountUpdatableCode,
+}
```

Comment:
Should we make this enum public and just have a single `account_type()` method on the `AccountId` struct? (instead of having 4 different methods `is_fungible_faucet()`, `is_non_fungible_faucet()` etc.).